specfile/tags.py

```diff
@@ -225,19 +239,18 @@ def __init__(
             raise ValueError(f"Invalid tag name: '{name}'")
         self.name = name
         self.value = value
-        self._expanded_value = expanded_value
         self._separator = separator
         self.comments = comments.copy()
         self._prefix = prefix or ""
         self._suffix = suffix or ""
+        self._context = context
 
     def __eq__(self, other: object) -> bool:
         if not isinstance(other, Tag):
             return NotImplemented
         return (
             self.name == other.name
             and self.value == other.value
-            and self._expanded_value == other._expanded_value
             and self._separator == other._separator
             and self.comments == other.comments
             and self._prefix == other._prefix
```

Comment:
The context doesn't need to be compared?

Reply:
The idea here (and in `SourcelistEntry.__eq__()`) was to be able to compare tags between different `Specfile` objects. But thinking about it, maybe it doesn't make a lot of sense. What do you think?

Anyway, this made me realize that the `_context` attribute needs to be be excluded from deepcopying, I'll fix that.